src/client/components/app/SliderCard/index.tsx

```diff
@@ -44,7 +44,16 @@ const StyledCardContent = styled(CardContent)`
   margin-bottom: 0;
   font-size: 1.6rem;
   line-height: 2.4rem;
+  padding-right: 12.8rem;
```

Comment:
I would remove this paddings (tablet and mobile too), looks weird. Maybe apply them in the custom banner when it is necessary.

![image](https://user-images.githubusercontent.com/78362532/167503818-85df29a7-3715-453d-b74a-cbaf361985b0.png)
![image](https://user-images.githubusercontent.com/78362532/167503840-e4aaeb18-ba93-4b88-be28-b8028297a173.png)

Reply:
![Uploading image.png…]()

Reply:
these need to be there for the default wide banners as the text should not go full width

<img width="759" alt="image" src="https://user-images.githubusercontent.com/80323528/167600301-93a2a6d8-ba00-4c6a-925b-d033d88405f1.png">

 - we are changing to the web-lib banners next week so this is prob ok to proceed with for now
 - the above look ok to me? and it fixes these: 
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/80323528/167600848-0dbe71f1-de74-47de-8950-1db814825465.png">

Reply:
okay.